core/state_processor_test.go

```diff
@@ -87,7 +87,7 @@ func u64(val uint64) *uint64 { return &val }
 // blockchain imports bad blocks, meaning blocks which have valid headers but
 // contain invalid transactions
 func TestStateProcessorErrors(t *testing.T) {
```

Comment:
Did we reorder sub-tests here? If so could we revert those to minimize the diff with Coreth?